moirae/models/ecm/utils.py

```diff
@@ -43,40 +43,49 @@ def _interp_func(self) -> Callable:
         return func
 
     def get_value(self, soc: Union[Number, List, np.ndarray]) -> np.ndarray:
-        """Computes value(s) at given SOC(s)
+        """
+        Computes value(s) at given SOC(s).
+
+        This function always returns a 3D array, of shape `(internal_batch_size, soc_batch_size, soc_dim)`, where
```

Comment:
This is great. Much simpler compared to the previous behavior, and the implementation is shorter too! Nice to be able to delete code 👏🏼

Reply:
Thank you! Yeah, it is definitely much easier to debug :D